projects/ngx-avatar/src/lib/sources/initials.ts

```diff
@@ -10,15 +10,15 @@ export class Initials implements Source {
 
   constructor(public sourceId: string) {}
 
-  public getAvatar(initialsSize: number): string {
-    return this.getInitials(this.sourceId, initialsSize);
+  public getAvatar(size: number): string | null {
+    return this.getInitials(this.sourceId, size);
   }
 
   /**
    * Returns the initial letters of a name in a string.
    */
   private getInitials(name: string, size: number): string {
-    name = name ? name.trim() : null;
+    name = name ? name.trim() : '';
```

Comment:
Should check for invalid name and return right away.

```suggestion
    if (!name) return '';
    name = name.trim();
```

And then the next check will change to check for empty string.

Reply:
This suggestion would introduce a double check of empty string. That does not look much cleaner to me. I'd rather not duplicate the logic of "we don't accept empty string, or whitespace-only strings"

Reply:
It is being checked two times already. Also, two times falsy check is slower than one falsy check plus one strict empty string comparison I believe, no need to say it's more readable.